javers-spring-boot-starter-mongo/src/main/java/org/javers/spring/boot/mongo/JaversMongoAutoConfiguration.java

```diff
@@ -62,13 +62,15 @@ public Javers javers() {
     }
 
     @Bean(name = "authorProvider")
+    @ConditionalOnMissingBean
```

Comment:
wydaje mi sie ze i tak mamy zależnośc na spring security. To mogłoby działac jak piszesz jesli zależnośc będzie Optional.